.gitlab-ci.yml

```diff
@@ -268,6 +268,16 @@ cargo-fmt:
     - cargo +nightly fmt --all -- --check
   allow_failure:                   true
 
+cargo-clippy:
+  stage:                           test
+  <<:                              *docker-env
+  <<:                              *test-refs
+  script:
+    - rustup +nightly component add clippy
```

Comment:
this should go to https://github.com/paritytech/scripts/blob/dedde1e7eccebc49007331567a24aaeff0600ec0/dockerfiles/ci-linux/Dockerfile#L28 to avoid the overhead by installing it every time, but also can be an issue to ci_cd.

Reply:
This seems to be in that docker file now. Thanks.